### ISSUE TITLE
Can't use umzug due to 'duplicate constraint name: "SequelizeMeta_name_key"'

### ISSUE THREAD

**Issue by HcgRandon** [cockroachdb/sequelize-cockroachdb#18]
Using umzug to handle migrations at runtime fails with this module.

It fails at the creation of the SequelizeMeta table.

The model they attempt to define can be found here:
https://github.com/sequelize/umzug/blob/master/src/storages/SequelizeStorage.js#L73

The error that is thrown: `SequelizeDatabaseError: duplicate constraint name: "SequelizeMeta_name_key"`

The raw query that is being executed:
```
Executing (default): CREATE TABLE IF NOT EXISTS "SequelizeMeta" ("name" VARCHAR(255) NOT NULL UNIQUE , UNIQUE ("name"), PRIMARY KEY ("name"));
```

Here is how init umzug for your testing:

```
  this.umzug = new Umzug({
   storage: 'sequelize',
   storageOptions: {
    sequelize: this.sequelize,
   },

   migrations: {
    params: [
     this.sequelize.getQueryInterface(),
     this.sequelize.constructor,
    ],
    path: this.opts.paths.migrations,
    pattern: /\.js$/,
   },
  });
```

Please let me know if you need any further information

**Comment by cuongdo** [comment]
assigning for triage

**Comment by vivekmenezes** [comment]
I tried

```
CREATE DATABASE named

SET DATABASE=named

CREATE TABLE IF NOT EXISTS "SequelizeMeta" ("name" VARCHAR(255) NOT NULL UNIQUE , UNIQUE ("name"), PRIMARY KEY ("name"));

CREATE TABLE IF NOT EXISTS "SequelizeMeta" ("name" VARCHAR(255) NOT NULL UNIQUE , UNIQUE ("name"), PRIMARY KEY ("name"));
```

against cockroachDB and it works. It's got to be something else.

**Comment by vivekmenezes** [comment]
Are you using a 1.0 release binary?

**Comment by HcgRandon** [comment]
I was using the docker image of cockroach, I don't know the exact versions but I can get them for you in a few hours. It's possible that I wasn't using the latest image.

**Comment by HcgRandon** [comment]
Using the latest docker image(v1.0.6) this issue can be replicated.

Simply
```
docker run -d -p 8080:8080 -p 26257:26257 cockroachdb/cockroach:v1.0.6
```
then open up the sql interface
```
docker exec -it [idfromabovecontainer] ./cockroach sql --insecure
```

then running your above commands fail. Here they are for reference:
```
root@:26257/> CREATE DATABASE named;
CREATE DATABASE

root@:26257/> SET DATABASE=named;
SET

root@:26257/named> CREATE TABLE IF NOT EXISTS "SequelizeMeta" ("name" VARCHAR(255) NOT NULL UNIQUE , UNIQUE ("name"), PRIMARY KEY ("name"));
pq: duplicate constraint name: "SequelizeMeta_name_key"

root@:26257/named> CREATE TABLE IF NOT EXISTS "SequelizeMeta" ("name" VARCHAR(255) NOT NULL UNIQUE , UNIQUE ("name"), PRIMARY KEY ("name"));
pq: duplicate constraint name: "SequelizeMeta_name_key"
```

**Comment by HcgRandon** [comment]
Also as #19 this issue is resolved in 1.1.0 builds

**Comment by benesch** [comment]
Yup, appears to be fixed by 1.1.0. Thanks for tracking this down! Closing this, as 1.1 will be released soon.
